src/main/java/com/github/manolo8/darkbot/Main.java

```diff
@@ -100,6 +100,12 @@
     public Main(StartupParams params) {
         super("Main");
         VerifierChecker.getAuthApi().setupAuth();
+
+        if (params.startConfig() == null)
+            configManager.loadConfig(null);
+        else
+            config = configManager.loadConfig(params.startConfig());
```

Comment:
This expression can be simplified to
```suggestion
        config = configManager.loadConfig(params.startConfig());
```